server/sonar-server/src/test/java/org/sonar/server/rule/ws/UpdateActionTest.java

```diff
@@ -310,6 +310,22 @@ public void fail_to_update_custom_when_description_is_empty() {
       .execute();
   }
 
+  @Test
+  public void throw_IllegalArgumentException_if_trying_to_update_builtin_rule_description() throws Exception {
+    logInAsQProfileAdministrator();
+    RuleDefinitionDto templateRule = db.rules().insert();
```

Comment:
name of the variable is not "templateRule" but "builtInRule" or only "rule"

Reply:
done